settings.gradle.kts

```diff
@@ -13,6 +13,6 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = "Polyfit"
+rootProject.name = "polyfit"
```

Comment:
fixing issue me & ben were having

Reply:
nit: make a separate pr for separate issue